packages/optimizer-docker/package.json

```
@@ -28,7 +28,8 @@
   },
   "homepage": "https://github.com/ampproject/amp-toolbox/tree/main/packages/optimizer",
   "dependencies": {
-    "@ampproject/toolbox-optimizer": "3.0.0-canary.1"
+    "@ampproject/toolbox-optimizer": "3.0.0-canary.1",
+    "prom-client": "^13.1.0"
```

One more thing! We prefer to pin dependencies and have a bot update them.

```suggestion
    "prom-client": "13.1.0"
```